contracts/LockedTokens.cdc

```diff
@@ -549,5 +560,8 @@ pub contract LockedTokens {
 
         self.LockedAccountCreatorStoragePath = /storage/lockedAccountCreator
         self.LockedAccountCreatorPublicPath = /public/lockedAccountCreator
+
+        /// create a single admin collection and store it
+        self.account.save(<-create TokenAdminCollection(), to: self.LockedTokenAdminCollectionStoragePath)
```

Comment:
What do you think about using the same pattern from `FlowToken`, where a separate administrator is passed into `init`? https://github.com/onflow/flow-core-contracts/blob/master/contracts/FlowToken.cdc#L168-L197

This would allow us to make the service account the token admin, which I think is what we want. The token admin needs to have access to the main token supply, which the service account already has.

Reply:
yeah I can do that. Do you know where the transaction template is that we use to deploy that? I'd rather reuse that then have to write a new one

Reply:
Here's how it's done for the `FlowToken`: https://github.com/onflow/flow-go/blob/master/fvm/bootstrap.go#L174-L181